interface/forms/care_plan/new.php

```diff
@@ -49,8 +49,9 @@
         <style>
             @media only screen and (max-width: 768px) {
                 [class*="col-"] {
-                width: 100%;
-                text-align:left !important;
+                    width: 100%;
+                    text-align:left !important;
```

Comment:
We should try to find a way to work around this instead of modifying bootstrap code here as this code could be messed up further down the line.

Reply:
I remove it and do the responsive testing, seem like the same.